### PR TITLE
Update URL and build for gtest formula

### PR DESCRIPTION
**Pull request by 4ndr3w**
Updated the gtest formula with the GitHub archive URL and generated makefile in the build process

Resolved issue #21 for me

### CODE REVIEW

File: gtest.rb [hunk 1]
```diff
@@ -2,9 +2,12 @@
...
   skip_clean "lib"
-  url 'http://googletest.googlecode.com/files/gtest-1.7.0.zip'
-  homepage 'http://code.google.com/p/googletest/'
-  sha256 '247ca18dd83f53deb1328be17e4b1be31514cedfc1e3424f672bf11fd7e0d60d'
+  url 'https://github.com/google/googletest/archive/release-1.7.0.zip'
+  homepage 'https://github.com/google/googletest'
+  sha256 'b58cb7547a28b2c718d1e38aee18a3659c9e3ff52440297e965f5edffe34b6d0'
+  
+  depends_on "automake"
+  depends_on "libtool"
```
Comment (wjwwood) [inline gtest.rb]: Should these have the `=> :build` suffix appended to them?

https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Formula-Cookbook.md#grab-the-url

**Comment by wjwwood** [comment]
Thanks for the pr! Lgtm other than a small comment.

**Comment by wjwwood** [comment]
Sweet, thanks!

**Comment by wjwwood** [comment]
Hmm, @4ndr3w does this formula work for you? I get this:

```
% brew install gtest
==> Installing gtest from ros/deps
==> Downloading https://github.com/google/googletest/archive/release-1.7.0.zip
Already downloaded: /Users/william/Library/Caches/Homebrew/gtest-1.7.0.zip
==> Patching
patching file include/gtest/gtest.h
==> glibtoolize
==> aclocal
Last 15 lines from /Users/william/Library/Logs/Homebrew/gtest/02.aclocal:
2016-09-08 15:27:10 -0700

aclocal

sh: autom4te: command not found
aclocal: error: echo failed with exit status: 127
```

**Comment by 4ndr3w** [comment]
Was just about to comment, adding the :build symbol appears to have broken it

**Comment by wjwwood** [comment]
Looks like we need a depend on autoconf too. I'll open a new pr.

**Comment by wjwwood** [comment]
Actually I just pushed it in ad038b7. Works for me now. Thanks!

**Pull request merged**